index.html

```diff
@@ -7,49 +7,57 @@
     <meta content="Custom ddnet based teeworlds client chillerbot-ux with warlist" name="description">
     <title>chillerbot-ux ddnet client</title>
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
```

Comment:
Not a big fan of external urls. Especially google. If you need a font it should be served by the site.